### ISSUE TITLE
Remove Twisted Flow from Trac

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4238]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#4238|
|Type|task|
|Created|2010-01-28 18:00:28Z|

The TwistedFlow wiki page can only be modified by users with wiki admin rights and it's probably a good idea to remove the page completely (I removed the link from TwistedProjects already) or update the page stating is has been removed.

Twisted Flow also still has an entry in the Trac components section, let's get rid of that as well?

Assigning it to somebody with admin rights.

Attachments:

* [flow.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/0ec/0ec5108ba89d4962c5721f5a41f06cba10c80429/6b218383b5e97a651cbac1a7be23a91590606d3d.patch) (637 bytes) - added by thijs on 2010-02-04 00:07:27Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4238 4238
type__task task
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1264701628000000 1264701628000000
changetime__1265746595000000 1265746595000000
version__None None
owner__ 
cc__thijs cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set status to `closed`|
|-|-|

I deleted the "flow" component (and the "foolscap" component).

I could delete the TwistedFlow page, destroying its content forever, irrecoverably.  Hm.  Actually there's not really anything interesting on the page anyway, so whatever, I'll do that.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra set status to `reopened`|
|-|-|

Found one more reference to the wiki page in the splitdoc, so reopening with a patch, a new ticket seems overkill..

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_4"></a>|@jesstess set owner to @thijstriemstra|
|-|-|

Looks good and dead, please apply.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [28140]) Apply flow.patch: Remove reference to old Twisted Flow project from the documentation.

Author: thijs
Reviewer: jesstess
Fixes: [#4238](https://github.com/twisted/twisted/issues/4238)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
